recipes/repos.rb

```diff
@@ -19,8 +19,8 @@
   baseurl     'https://revenants.jfrog.io/artifactory/rpm-local/$releasever/os/$basearch/'
   description 'RevDB repository'
   gpgcheck    false
-  username(lazy { node.run_state['artifactory_user'] })
-  password(lazy { node.run_state['artifactory_api_key'] })
+  username lazy { node.run_state['artifactory_user'] }
```

Comment:
I'm curious why did you remove the parenthesis? I recall RubyCop made me add them.